frontend/src/App.js

```diff
@@ -12,25 +13,36 @@ import AttendeeList from "./attendees/AttendeeList";
 import LocationUpdate from "./admin/LocationUpdate";
 import AddLocation from "./admin/AddGroup";
 import LoginPage from "./LoginPage";
+import SignIn from "./authentication/SignIn";
+import SignUp from "./authentication/SignUp";
+import PasswordReset from "./authentication/PasswordReset";
```

Comment:
`PasswordReset` doesn't seem to exist and is throwing an error. There is no file named like that in the authentication folder